eip-0001.md

```diff
@@ -120,14 +120,29 @@ Specification: List Of All Proposed Endpoints
 - GET: `/scan/boxesUnspent/{scanId}` - List all boxes that have been tracked by the scan and are still unspent/part of the UTXO set
 
 
-Specification: Simple Wallet Scan
+Specification: List Of Implemented Endpoints
+--------------------------------------------
+
+- POST: `/scan/register` - 3.3.0
+- POST: `/scan/deregister` - 3.3.0
+- POST: `/scan/stopTracking/{scanId}/{boxId}` - 3.3.0
+- GET: `/scan/listAll` - 3.3.0
+- GET: `/scan/boxesUnspent/{scanId}` - 3.3.0
+
+Objects and enpoints description can be found in openapi.yaml file in [Ergo protocol reference client repository](https://github.com/ergoplatform/ergo).
+
```

Comment:
Since the main purpose of EIP is to be used by app, the semantics of each API method should be fully specified here (in a self sufficient manner).

It is better to put precise specification of each method here (since the reference implementation already finalized).

After publication, this specification will serve as baseline truth both for apps and future releases.

Reply:
I guess a link to openapi.yaml would be enough